packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -106,10 +107,9 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
 			throw new Error("A new or existing file must be specified to create container!");
 		}
 
-		const protocolSummary = createNewSummary?.tree[".protocol"];
-		if (protocolSummary) {
+		if (isCombinedAppAndProtocolSummary(createNewSummary)) {
 			const documentAttributes = getDocAttributesFromProtocolSummary(
-				protocolSummary as ISummaryTree,
+				createNewSummary.tree[".protocol"],
```

Comment:
Is it safe to always ask for combined app protocol summary? In this case only protocol summary is required. I wonder if there is a case where protocol is there but app is not

Reply:
this is in the create new path, so it will always be a combined summary